nautobot/docs/rest-api/overview.md

```diff
@@ -785,3 +785,478 @@ http://nautobot/api/dcim/sites/ \
+        "last_updated": "2023-04-12T19:25:51.413824Z",
+        "name": "Active",
+        "color": "4caf50",
+        "description": "Unit is active"
+    },
+    "parent": {
+        "id": "0e19e475-89c9-4cf4-8b5f-a0589f0950cd",
+        "display": "Campus-01",
+        "url": "http://nautobot/api/dcim/locations/0e19e475-89c9-4cf4-8b5f-a0589f0950cd/",
+        "custom_fields": {
+            "example_plugin_auto_custom_field": null
+        },
+        "notes_url": "http://nautobot/api/dcim/locations/0e19e475-89c9-4cf4-8b5f-a0589f0950cd/notes/",
+        "tree_depth": null,
+        "time_zone": "Asia/Baghdad",
+        "created": "2023-04-12T19:29:06.884754Z",
+        "last_updated": "2023-04-12T19:29:06.906503Z",
+        "name": "Campus-01",
+        "slug": "campus-01",
+        "description": "Local take each compare court exactly.",
+        "facility": "328",
+        "asn": null,
+        "physical_address": "",
+        "shipping_address": "",
+        "latitude": null,
+        "longitude": "104.200000",
+        "contact_name": "Frances Hernandez",
+        "contact_phone": "",
+        "contact_email": "",
+        "comments": "Sort share road candidate.",
+        "status": "28eb334b-4171-4da4-a03a-fa6d0c6a9442",
+        "parent": null,
+        "location_type": "e3d4a9af-c6c1-4582-b483-a13301eb6e28",
+        "tenant": "5b1feadb-fab0-4f81-a53f-5192d83b0216",
+        "tags": [
+            "6e8ce6c9-0a8c-4731-b02a-b2ec19db5c52",
+            "1e986e5d-099d-4742-bfa5-16363abea5fd",
+            "21afbdad-c782-4fc5-9b3b-7b59273bb08c",
+            "0118b994-9943-4295-946b-92b27efe15db"
+        ]
+    },
+    "location_type": {
+        "id": "4edcc111-e3f7-4309-ab0e-eb34c001874e",
+        "display": "Campus → Building",
+        "url": "http://nautobot/api/dcim/location-types/4edcc111-e3f7-4309-ab0e-eb34c001874e/",
+        "custom_fields": {},
+        "notes_url": "http://nautobot/api/dcim/location-types/4edcc111-e3f7-4309-ab0e-eb34c001874e/notes/",
+        "tree_depth": null,
+        "content_types": [
+            "dcim.rack",
+            "ipam.prefix",
+            "ipam.vlan",
+            "dcim.rackgroup",
+            "ipam.vlangroup"
+        ],
+        "created": "2023-04-12T19:29:06.707759Z",
+        "last_updated": "2023-04-12T19:29:06.716482Z",
+        "name": "Building",
+        "slug": "building",
+        "description": "Protect growth bill all hair along.",
+        "nestable": false,
+        "parent": "e3d4a9af-c6c1-4582-b483-a13301eb6e28"
+    },
+    "tenant": null,
+    "tags": []
+}
+```
+
+### ?depth=2 and beyond
+
+For `TreeModels`, sometimes it is necessary to traverse up more than one tree level and to retrieve an ancestor's information that is not a direct parent of the object.
+In this case, `?depth=2` and beyond would become very useful.
+
+!!! note
+    Notice that the location and its location_type's parent's parent's serializers are shown as well.
```

Comment:
I find this sentence difficult to parse but I don't have an immediate alternative to suggest.